### ISSUE TITLE
How can I move macros popup

### ISSUE THREAD

**Issue by neoncobra** [gingerbeardman/Macro#3]
Hi, thanks for your extension, been looking for something like this for a long time. 

Could you please tell me how to move the extention's popup to the bottom corner or outside of the edited file.
Now it's inconvenient because the popup covers the Find/Replace menu buttons at the top of the editors window
![2024-11-16_03-32](https://github.com/user-attachments/assets/860961af-19ae-46ab-9fbf-5e9d2b822df3)

**Comment by gingerbeardman** [comment]
Yes, it's an issue. Maybe I could add a hide button, but then there would need to be a way to bring it back as it's the only way to stop the recording.

I'll give it some thought.
